### PR TITLE
Create README.md for TTW Organisation

### PR DESCRIPTION
**Pull request by aleesteele**
Tied to this issue: https://github.com/the-turing-way/the-turing-way/issues/3282

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+## Welcome to _The Turing Way_ 👋
+
+_The Turing Way_ is open source and community-led handbook for reproducible, ethical and collaborative data science.
```
Comment (aleesteele) [inline profile/README.md:3]: We've also used the phrase "The Turing Way project is an open source community that documents, maintains, and translates best practices in data science and AI.", but this one above is most widely used!

File: profile/README.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+## Welcome to _The Turing Way_ 👋
```
Comment (JimMadge) [inline profile/README.md:1]:
```suggestion
# Welcome to _The Turing Way_ 👋
```

**Review by JimMadge — APPROVED** [review]
Looks great.

Just a suggestion to change the heading level.

File: profile/README.md (hunk 2 above)
Comment (aleesteele) [reply profile/README.md]: Thank you @JimMadge 🙏

**Pull request merged**